src/Generator.Bind/LicenseResource.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using System.Reflection;
+
+namespace Bind
+{
+    /// <summary>
+    /// Class for handling embedded license text.
+    /// </summary>
+    internal class LicenseResource
```

Comment:
In the case of having embedded resources accessible, it's typically delegated to a class made especially for embedded resources. Maybe we could generalize this to just EmbeddedResources?

Reply:
Sounds good, will fix it.